charts/flyte-core/values.yaml

```diff
@@ -975,6 +975,8 @@ configmap:
         - "metadata"
         - "admin"
       eventVersion: 2
+      injectUserAnnotations: false
+      userAnnotationPrefix: "flyte.ai"
```

Comment:
Could we define something like these
```
injectUserAnnotations: false
userAnnotationPrefix: "flyte.ai"
userAnnotationKeys:
  - email
  - sub
  ```
  And allow the following suffixes and use that populate those info.
  A lot of case the subject identifier can be useful to implement more authz policies based on the unique identifier if such a system exists.